submissions/NwagboJohn/php_task/my_php

```diff
@@ -0,0 +1 @@
+php
```

Comment:
Please remove this file as it's not needed

Reply:
It has been deleted